html/elements/body.json

```diff
@@ -141,19 +141,17 @@
               },
               "oculus": "mirror",
               "opera": {
-                "version_added": true
+                "version_added": "≤12.1"
```

Comment:
The codepen doesn't load in Opera 12.16, but I pasted it into http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=10877 and it's not supported on all elements.